models/event/event.go

```diff
@@ -453,7 +453,12 @@ func BuildSoftPartitionSuffix(
 		if distance == 0 {
 			// Same partition, use base suffix
 		} else if distance < 0 {
-			return "", fmt.Errorf("partition time %v for column %q is in the future of execution time %v", partitionColumnValue, sp.PartitionColumn, executionTime)
+			slog.Warn(
```

Comment:
Is there a point in flooding our logs if this is expected? Also can we add a future timestamp test case?